src/main/java/uk/gov/hmcts/reform/sendletter/config/ReportsServiceConfig.java

```diff
@@ -2,20 +2,51 @@
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 
+import static java.util.stream.Collectors.toMap;
+
 @ConfigurationProperties(prefix = "reports")
 public class ReportsServiceConfig {
 
     private Map<String, String> serviceConfig;
 
+    public static class Mapping {
```

Comment:
@rkondratowicz We need this class. The config was loaded in a different format without this class
`Eg:` The below config is loaded as 2 entries
```
[0].service=some_service_name
[0].display-name=some service
```

Reply:
It was not used in the setter before (or anywhere else)